downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc

```diff
@@ -0,0 +1,22 @@
+[id="controller-obtaining-subscriptions"]
+
+= Obtaining an authorized Ansible {ControllerName} subscription
+
+If you already have a subscription to a Red Hat product, you can acquire an automation controller subscription through that subscription. 
+If not, you can request a link:https://www.redhat.com/en/technologies/management/ansible/try-it[trial subscription].  
+
+.Procedure
+
+* If you already have a {PlatformName} subscription, use your Red Hat customer credentials when you launch the controller to access your subscription information. 
+See xref:controller-importing-subscriptions[Importing a subscription].
+
+* If you have a non-Ansible Red Hat or Satellite subscription, access the automation controller with one of these methods:
+** Enter your username and password on the license page.
+** Obtain a subscriptions manifest from the link:https://access.redhat.com/management/subscription_allocations[Subscription Allocations] page on the Red Hat Customer Portal. 
+For detailed information, see link:http://docs.ansible.com/automation-controller/4.4/html/userguide/import_license.html#obtain-sub-manifest[Obtaining a subscriptions manifest] in the Automation Controller User Guide.
+** If you do not have a {PlatformName} subscription, go to link:https://www.redhat.com/en/technologies/management/ansible/trial[Try Red Hat Ansible Automation Platform] and request a trial subscription.
+
+.Additional resources
+
+To understand what is supported with your subscription, see link:http://docs.ansible.com/automation-controller/4.4/html/userguide/license-support.html#licenses-feat-support[Red Hat Ansible Automation Platform Controller Licensing, Updates, and Support]. 
```

Comment:
There's an xref for this now. But probably better to wait until the UG is complete.